db/seeds/run-seed-test.js

```diff
@@ -0,0 +1,10 @@
+const data = require("../data/test-data")
+process.env.NODE_ENV = "test"
+const seed = require("./seed")
+const db = require("../connection.js");
+
+const runSeed = () => {
+    return seed(data).then(() => db.end());
```

Comment:
Your test file will seed the test data in the `beforeEach` you have so this shouldn't be necessary